web/Services/AdminService.cs

```diff
@@ -44,18 +44,23 @@ private async Task<List<UserDbModel>> GetCurrentAdmins()
 
     private async Task<List<UserDbModel>> GetPendingAdmins()
     {
-        List<UserDbModel> PendingAdmins = await _dbContext.Users.Where(u => u.IsAdmin == false && u.HasRequestedAdmin).ToListAsync();
+        List<UserDbModel> PendingAdmins =
+            await _dbContext.Users.Where(u => u.IsAdmin == false && u.HasRequestedAdmin).ToListAsync();
```

Comment:
I'd indent this differently
```suggestion
        List<UserDbModel> PendingAdmins = await _dbContext.Users.Where(u => 
            u.IsAdmin == false && 
            u.HasRequestedAdmin)
            .ToListAsync();
```